docs/docs/genai/eval-monitor/scorers/llm-judge/agentic-overview.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Agentic Judges Overview
+sidebar_label: Overview
+description: Understanding how judges become autonomous agents for deep trace analysis
+---
+
+import FeatureHighlights from "@site/src/components/FeatureHighlights";
+import ConceptOverview from "@site/src/components/ConceptOverview";
+import TilesGrid from "@site/src/components/TilesGrid";
+import TileCard from "@site/src/components/TileCard";
+import { Brain, Search, Activity, GitBranch, Target, Shield, FileText } from "lucide-react";
+
+# Agentic Judges: Autonomous Trace Analysis
+
+Agentic Judges represent a paradigm shift in LLM evaluation. Instead of simply assessing inputs and outputs, these judges act as **autonomous agents** equipped with tools to investigate your application's execution in depth.
+
+## What Makes a Judge "Agentic"?
+
+When you use the `{{ trace }}` template variable in your judge instructions, MLflow transforms your judge from a passive evaluator into an active investigator. The judge gains access to MCP (Model Context Protocol) tools that interface with MLflow's APIs, enabling it to:
+
+<FeatureHighlights features={[
+  {
+    icon: Search,
+    title: "Explore Execution Flows",
+    description: "Navigate through traces, fetching specific spans as needed to understand the application's behavior."
+  },
+  {
+    icon: Activity,
+    title: "Analyze Performance",
+    description: "Examine timing, latency, and resource usage across different components of your application."
+  },
+  {
+    icon: GitBranch,
+    title: "Detect Patterns",
+    description: "Identify bottlenecks, redundancies, circular logic, and inefficient execution paths."
+  },
+  {
+    icon: Shield,
+    title: "Verify Behavior",
+    description: "Check tool usage, error handling, retry logic, and compliance with expected patterns."
+  }
+]} />
+
+## Field-Based vs Agentic Evaluation
```

Comment:
@BenWilson2 can we include ease of use in here too? It's much easier to define agent-as-a-judge